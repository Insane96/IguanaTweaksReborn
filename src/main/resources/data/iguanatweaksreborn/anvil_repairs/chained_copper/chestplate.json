{
	"item_to_repair": {
		"id": "iguanatweaksreborn:chained_copper_chestplate"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:copper_ingot"
			},
			"amount": 6
		}
	]
}