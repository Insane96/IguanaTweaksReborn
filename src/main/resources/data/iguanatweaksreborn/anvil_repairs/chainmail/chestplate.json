{
	"item_to_repair": {
		"id": "minecraft:chainmail_chestplate"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:chain"
			},
			"amount": 5
		}
	]
}