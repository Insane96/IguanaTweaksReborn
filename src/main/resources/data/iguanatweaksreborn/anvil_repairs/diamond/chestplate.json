{
	"item_to_repair": {
		"id": "minecraft:diamond_chestplate"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:diamond"
			},
			"amount": 6
		}
	]
}