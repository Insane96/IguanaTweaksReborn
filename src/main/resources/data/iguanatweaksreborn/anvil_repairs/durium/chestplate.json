{
	"item_to_repair": {
		"id": "iguanatweaksreborn:durium_chestplate"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:iron_ingot"
			},
			"amount": 8,
			"max_repair": 0.5
		},
		{
			"repair_material": {
				"id": "iguanatweaksreborn:durium_ingot"
			},
			"amount": 2
		},
		{
			"repair_material": {
				"id": "iguanatweaksreborn:durium_nugget"
			},
			"amount": 18
		}
	]
}