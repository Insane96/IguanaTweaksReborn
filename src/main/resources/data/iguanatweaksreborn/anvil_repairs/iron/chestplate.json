{
	"item_to_repair": {
		"id": "minecraft:iron_chestplate"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:iron_ingot"
			},
			"amount": 6
		},
		{
			"repair_material": {
				"id": "minecraft:iron_nugget"
			},
			"amount": 58,
			"cost_multiplier": 9
		}
	]
}