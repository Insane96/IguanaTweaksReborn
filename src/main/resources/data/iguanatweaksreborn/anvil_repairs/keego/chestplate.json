{
	"item_to_repair": {
		"id": "iguanatweaksreborn:keego_chestplate"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:diamond"
			},
			"amount": 8,
			"max_repair": 0.5
		},
		{
			"repair_material": {
				"id": "iguanatweaksreborn:keego"
			},
			"amount": 2
		}
	]
}