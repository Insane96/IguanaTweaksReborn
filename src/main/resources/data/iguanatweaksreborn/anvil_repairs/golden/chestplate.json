{
	"item_to_repair": {
		"id": "minecraft:golden_chestplate"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:gold_ingot"
			},
			"amount": 8
		},
		{
			"repair_material": {
				"id": "minecraft:gold_nugget"
			},
			"amount": 72,
			"cost_multiplier": 9
		}
	]
}